testnet/tools/nns-tools/lib/nns_neurons.sh

```diff
@@ -30,3 +30,32 @@ nns_claim_or_refresh() {
         }
       )"
 }
+
+
+
+##: nns_list_my_neurons
+## Usage: $1 <NNS_URL>
+## List the neurons owned by the current dfx identity
+nns_list_my_neurons() {
```

Comment:
This seems pretty tangential...

Reply:
It is.  It was used in testing proposals